src/Cake.Common/Tools/DotNetBuildSettingsExtensions.cs

```diff
@@ -31,6 +31,25 @@ public static DotNetBuildSettings WithTarget(this DotNetBuildSettings settings,
         }
 
         /// <summary>
+        /// Adds .NET build targets to the configuration.
+        /// </summary>
+        /// <param name="settings">The settings.</param>
+        /// <param name="targets">The .NET build targets.</param>
+        /// <returns>The same <see cref="DotNetBuildSettings"/> instance so that multiple calls can be chained.</returns>
+        public static DotNetBuildSettings WithTarget(this DotNetBuildSettings settings, params string[] targets)
```

Comment:
Would prefer if we took a `IEnumerable<string>` here instead of params. Also, the method should be called `WithTargets` (plural).